src/components/Settings/index.tsx

```diff
@@ -98,10 +110,19 @@ export default function SettingsTab() {
     if (!open) setIsSelectingLanguage(false)
   }, [open])
 
+  const { mixpanelHandler } = useMixpanel()
+  const setShowTutorialSwapGuide = useTutorialSwapGuide()[1]
+  const openTutorialSwapGuide = () => {
+    setShowTutorialSwapGuide({ show: true, step: 0 })
+    mixpanelHandler(MIXPANEL_TYPE.TUTORIAL_CLICK_START)
+    toggle()
+  }
```

Comment:
`useCallback`

Reply:
I dont see any performance problems here.

Reply:
You left many place without `useCallback`. Oneday one of them have performance issue and someone have to trace back every of your missed. Should add `useCallback` when posible. Also it help mantainer who read your code know that these function is protected guarantee by `useCallback`, not having to check if it's have issue or not.

Reply:
Yeah no problem at all, Nam should tell a bit more about why

Reply:
any update on this?

Reply:
... update what? You guys not seeing my message?

Reply:
In that case, that function is used only for onClick.
No need memoize it.
We will memorize it if needed. 
The more we use, the more Ram it costs.